docs/check_links.sh

```diff
@@ -47,6 +48,37 @@ echo
 echo "Working in ${MY_DIR} folder"
 echo
 
+if [[ ! -f _build/html/index.html ]]; then
+   echo "You should build documentation first."
+   echo
+   echo "If you use the breeze environment then you can do it using the following command:"
+   echo "./breeze --build-docs"
+   echo
+   exit 1
+fi
+
+if [[ -f /.dockerenv ]]; then
```

Comment:
I think we should only run it in docker container. There is no need to run it outside of it and readarray is not working for vast majority of our users (from the experience vast majority of Airflow developers use MacOS and readarray won't work there). 

Let's keep to "standard" development environment we already have - Breeze + docker. I think this should be explicitly "in_container" script and it should fail if run outside of it.

Doing this we also encourage people to use it.